Cargo.toml

```diff
@@ -50,6 +50,8 @@ maptos-dof-execution = { path = "protocol-units/execution/dof" }
 maptos-opt-executor = { path = "protocol-units/execution/opt-executor" }
 maptos-fin-view = { path = "protocol-units/execution/fin-view" }
 maptos-execution-util = { path = "protocol-units/execution/util" }
+# infra 
```

Comment:
```suggestion
## infra
```
Otherwise it looks like a "first-level header" under which "mempool", "sequencing", etc. are grouped.